tests/performance/loadtest/loadtest_test.go

```diff
@@ -136,12 +137,18 @@ func PullImageInAllEdgeNodes(appDeployments []string) {
 }
 
 //Run Test cases
-var _ = Describe("Application deployment test in Perfronace test EdgeNodes", func() {
+var _ = Describe("Application deployment test in Performance test EdgeNodes", func() {
```

Comment:
Could you pls cleanup all those typos in a separate commit?

Reply:
yeah, i'll do that.